packages/onboarding/src/setup-tailored-site-after-creation.ts

```diff
@@ -93,13 +93,37 @@ export function setupSiteAfterCreation( { siteId, flowName }: SetupOnboardingSit
 		return Promise.resolve();
 	};
 
+	const removeInitialPost = async ( siteId: number, flowName: string ) => {
+		if ( flowName === NEWSLETTER_FLOW ) {
+			// We are deleting the initial "Hello World!" post for newsletter sites
+			// The id for the first post on a new site will be 3
+
+			// First request will move post to trash
+			await wpcomRequest( {
+				path: `/sites/${ siteId }/posts/3/delete?http_envelope=1`,
+				method: 'POST',
+				apiNamespace: 'rest/v1.1',
+			} );
+
+			// Second request will permanently delete the post
+			await wpcomRequest( {
+				path: `/sites/${ siteId }/posts/3/delete?http_envelope=1`,
+				method: 'POST',
+				apiNamespace: 'rest/v1.1',
+			} );
```

Comment:
If the post of id 3 isnt available, will this fail gracefully?

Reply:
We need a try/catch here as @Sharpirate suggested too in case the post id isn't 3 or if there's a failure.